include/grl/kuka/KukaJAVAdriver.hpp

```diff
@@ -495,14 +533,22 @@ namespace grl { namespace robot { namespace arm {
        armState_.clearCommands();
       std::copy(range,armState_.commandedCartesianWrenchFeedForward);
     }
-    
+
    /// @todo should this exist? is it written correctly?
    void get(KukaState & state)
    {
      boost::lock_guard<boost::mutex> lock(jt_mutex);
      state = armState_;
    }
-   
+
+   void getWrench(KukaState & state)
```

Comment:
and yes, it is with respect to end effector... Should have written cartesian wrench